build-tests/api-documenter-scenarios/src/runScenarios.ts

```
@@ -1,154 +1,85 @@
 
-    // TODO: Ensure that the checked-in files are up-to-date
-    // Run the API Documenter command-line
-    executeCommand(
-      'node node_modules/@microsoft/api-documenter/lib/start ' +
-        `generate --input-folder etc/${scenarioFolderName} --output-folder etc/${scenarioFolderName}/markdown`
-    );
-  }
-
-  // Delete the transient `api-documenter.json` file before completing, as it'll just be whatever the last scenario
-  // was, and shouldn't be committed.
-  FileSystem.deleteFile(apiDocumenterJsonPath);
-
-  if (!anyErrors) {
-    process.exitCode = 0;
+        const apiDocumenterJson: {} = {
+          $schema: 'https://developer.microsoft.com/json-schemas/api-extractor/v7/api-documenter.schema.json',
+          outputTarget: 'markdown',
+          tableOfContents: {},
+          ...apiDocumenterJsonOverrides
+        };
+
+        await JsonFile.saveAsync(apiDocumenterJson, apiDocumenterJsonPath, { ensureFolderExists: true });
+
+        // TODO: Ensure that the checked-in files are up-to-date
+        // Run the API Documenter command-line
+        const childProcess: ChildProcess = Executable.spawn(
+          process.argv0,
+          [
+            'node_modules/@microsoft/api-documenter/lib/start',
+            'generate',
+            `--input-folder`,
+            `temp/etc/${scenarioFolderName}`,
+            '--output-folder',
+            `temp/etc/${scenarioFolderName}/markdown`
+          ],
+          {
+            stdio: 'inherit'
+          }
+        );
+
+        await Executable.waitForExitAsync(childProcess, {
+          throwOnNonZeroExitCode: true,
+          throwOnSignal: true
+        });
+      }
+    });
+  } finally {
+    // Delete the transient `api-documenter.json` file before completing, as it'll just be whatever the last scenario
+    // was, and shouldn't be committed.
+    await FileSystem.deleteFileAsync(apiDocumenterJsonPath);
```

Wrap this in a try-catch to not swallow if its a not-exists error. Keeps logs clean in failures earlier on in the build process

---

The default is to not throw if it already doesn't exist.